com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariablePermission.cs

```diff
@@ -3,26 +3,29 @@ namespace Unity.Netcode
     /// <summary>
     /// Permission type
     /// </summary>
-    public enum NetworkVariablePermission
+//    public enum NetworkVariableWritePermission
+//    {
+//        /// <summary>
+//        /// Server-only operation
+//        /// </summary>
+//        ServerOnly,
+//
+//        /// <summary>
+//        /// Owner-ownly
+//        /// </summary>
+//        OwnerOnly,
+//    }
+
+    public enum NetworkVariableReadPermission
     {
         /// <summary>
         /// Everyone
         /// </summary>
         Everyone,
```

Comment:
Not sure if I understand this. Why are we keeping `Everyone` and `OwnerOnly`. I thought that the decision was that we only support `ServerOnly` and remove all others.

Reply:
Ah I see this is read only now, makes sense